enterprise/NOTICE.txt

```diff
@@ -1,29 +1,26 @@
 Neo4j
 Copyright © 2002-2016 Network Engine for Objects in Lund AB (referred to
-in this notice as “Neo Technology”)
+in this notice as "Neo Technology")
    [http://neotechnology.com]
 
 This product includes software ("Software") developed by Neo Technology.
 
-The copyright in the bundled Neo4j graph database (including the
-Software) is owned by Neo Technology. The Software developed and owned
-by Neo Technology is licensed under the GNU AFFERO GENERAL PUBLIC
-LICENSE Version 3 (http://www.fsf.org/licensing/licenses/agpl-3.0.html)
-("AGPL") to all third parties and that license, as required by the AGPL,
-is included in the LICENSE.txt file.
+The software ("Software") is developed and owned by Network Engine
```

Comment:
Should this be part of a security PR? Perhaps it should be at least in a separate commit?